app/pages/solutions-list/controller.js

```diff
@@ -16,20 +16,24 @@ const transformCapabilities = ({ capabilitiesSelected }) => {
 };
 
 export const getSolutionListPageContext = async ({ filterType }) => {
-  const solutionListManifest = getSolutionListManifest(filterType);
-  const endpoint = getEndpoint({ endpointLocator: 'getSolutionListData', options: { filterType } });
+  let filter = filterType;
+  if (filter === 'dfocvc') filter = 'dfocvc001';
```

Comment:
Is it possible to use `'dfocvc001'` to avoid this?

Reply:
It would be nicer for coding to be that yes, the url which the end use see's would have to be changed to /Solutions/dfocvc001

Will this be accepted?

Reply:
From out point-of-view, I think that's fine. We usually treat URLs as an implementation detail. If someone decides they don't like it we can change it later, although I think that's unlikely.